jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildDockerTask.java

```diff
@@ -127,6 +128,12 @@ public void buildDocker()
               + ex.getInvalidPathValue(),
           ex);
 
+    } catch (InvalidPlatformException ex) {
+      throw new GradleException(
+          "from.platforms contains a platformConfiguration that is missing required fields or has invalid values: "
```

Comment:
So, I see we have a difference from other `Invalid*Exception` in that we provide better exception messages when instantiating `InvalidPlatformException`. Then we are not making use of the exception message. In this case, I think we can provide a more helpful and detailed message by

"from.platforms contains a platform configuration that is missing required values or has invalid values: " + ex.getMessage() + ": " + ex.getInvalidPlatform()

Note "platformConfiguration" --> "platform configuration". The message will be shown to the end user, and `platformConfiguration` is an internal Java variable name in our code that the user doesn't know about. Similarly, I think "required values" is more appropriate for the end user.